app/views/layouts/_navbar.html.erb

```diff
@@ -0,0 +1,24 @@
+<nav class="nav">
+      <%= link_to 'Friends App', root_path %>
+      <div class="menu">
+        <div class="menu-item">
+          <%= menu_link_to 'Timeline', posts_path %>
+        </div>
+        <div class="menu-item">
+          <%= menu_link_to 'All users', users_path %>
+        </div>
+        <div class="menu-item">
+          <%= menu_link_to current_user.name, users_path if signed_in? %>
+        </div>
+        <div class="menu-item">
+          <%= menu_link_to current_user.friends.count, users_path if signed_in? %>
+        </div>
+      </div>
+      
+      <% if current_user %>
+        <%= link_to 'Sign out', destroy_user_session_path, method: :delete %>
+      <% else %>
+        <%= link_to 'Sign in', user_session_path %>
```

Comment:
- [ ] [OPTIONAL] It is a good practice to have zero logic in any of the views directory files. Create methods for each logic statement in the helpers directory and call those methods instead